.gitignore

```diff
@@ -7,3 +7,4 @@
 /tmp
 /FEATURES.md
 /dev-scripts/
+.vscode/launch.json
```

Comment:
If we're going to add this, shouldn't we just exclude the whole `.vscode` folder?

Reply:
fixed